core/src/main/java/com/yahoo/imapnio/async/request/ListStatusCommand.java

```diff
@@ -0,0 +1,129 @@
+package com.yahoo.imapnio.async.request;
+
+import java.nio.charset.StandardCharsets;
+
+import javax.annotation.Nonnull;
+
+import com.sun.mail.imap.protocol.BASE64MailboxEncoder;
+import com.yahoo.imapnio.async.exception.ImapAsyncClientException;
+
+import io.netty.buffer.ByteBuf;
+import io.netty.buffer.Unpooled;
+
+/**
+ * This class defines IMAP LIST-STATUS Command, RFC5819. This extension is part of LIST-EXTENDED extension, RFC5258.
+ *
+ * This command is specialized to support LIST-STATUS. It does not support list-select-opts, nor other RETURN options.
+ *
+ * @see <a href="https://tools.ietf.org/html/rfc5819#page-2" target="_blank">LIST-STATUS extension</a>
+ * @see <a href="https://tools.ietf.org/html/rfc5258#page-19" target="_blank">LIST-EXTENDED extension</a>
+ *
+ *      <blockquote>
+ *
+ *      <pre>
+ *
+ * list =      "LIST" [SP list-select-opts] SP mailbox SP mbox-or-pat
```

Comment:
Do you not want to support multiple mailbox patterns ?

Reply:
done.  Thanks for your valuable suggestion.